XenModel/Actions/VMSS/AssignVMsToPolicyAction.cs

```diff
@@ -86,15 +87,15 @@ public RemoveVMsFromPolicyAction(VMSS policy, List<XenRef<VM>> selectedVMs)
             _policy = policy;

```

Comment:
I'll remove the unused `_policy`. We could delete `RemoveVMsFromPolicyAction` and `RemoveVMsFromVMApplianceAction` since the body of Run() is not dissimilar, however, afterwards we'd need some `if/else` logic in the `AssignVMsFromPolicyAction` and `AssignVMsFromVMApplianceAction` to set the action title and description for the remove case properly, so we may as well leave them as they are, they are not too big anyway.